### ISSUE TITLE
Plugin not starting when GOG Galaxy launches

### ISSUE THREAD

**Issue by fl4shback** [urwrstkn8mare/gog-riot-integration#13]
Hi!

For some reason, the plugin isn't starting like all the other plugins when Galaxy starts.
I have to go in settings and press connect at each Galaxy restart which kind of deafeats the purpose of the integration.
Is this normal ?
Tried on a fresh install of Galaxy and of the plugin.

ClientLog:
[GalaxyClient.log](https://github.com/urwrstkn8mare/galaxy-riot-integration/files/5684679/GalaxyClient.log)

From what I can see, the plugin is detected but for some reason Galaxy doesn't seem to try to start it..

**Comment by urwrstkn8mare** [comment]
Hi @fl4shback that is not the intended behaviour. I will have a look at the log and try to fix this issue for you tomorrow as I have to sleep soon.

Thanks for letting me know.

**Comment by fl4shback** [comment]
Hey @urwrstkn8mare !
Did you end up finding anything ?

**Comment by urwrstkn8mare** [comment]
Sorry for the wait, I've been pretty busy this time of year. I've just had a look at the client log but were you able to get the log file for the riot integration? It can be found in `%programdata%\GOG.com\Galaxy\logs` and will have `riot` in its name.

**Comment by fl4shback** [comment]
The thing is, the riot plugin isn't starting until I force it to.
So, no logs until I force it to start.

I just noticed something strange in the GalaxyClient.log though made a smaller version and attached it under the name GalaxyClient_Extract.log

For some reason there seems to be an issue with storing plugin status in the database (maybe I'm an isolated case of DB corruption who knows)

Logs attached:

[GalaxyClient.log](https://github.com/urwrstkn8mare/galaxy-riot-integration/files/5790608/GalaxyClient.log)
[plugin-riot-b4c0c2f5-23dc-4326-aa65-e9fabef61a3b.log](https://github.com/urwrstkn8mare/galaxy-riot-integration/files/5790609/plugin-riot-b4c0c2f5-23dc-4326-aa65-e9fabef61a3b.log)
[GalaxyClient_Extract.log](https://github.com/urwrstkn8mare/galaxy-riot-integration/files/5790610/GalaxyClient_Extract.log)

**Comment by urwrstkn8mare** [comment]
Yes something is definitely up with GOG Galaxy and its DB. Have you tried disconnecting the plugin and reconnecting? Like disconnecting, quitting, restarting, connecting. I'll try searching for some solution for this but that's the best advice I can give right now sorry. I'll let you know what I find.

**Comment by fl4shback** [comment]
So, little follow up on this if anyone ever have this issue one day.

I fired up SQLite DB Browser and removed everything riot linked in my database. Seems to have solved my issue.

If I remember correctly, the rockstar plugin used to use the riot platform ID in galaxy (before galaxy team added rockstar as a platform). That's probably where my issue was coming from.

Tried to restart Galaxy 2-3 times since I did this and it seems to work reliably now.
